### DISCUSSION TITLE
How to disable/enable user text input when there are buttons in the conversation BotMan ?

### DISCUSSION THREAD

**Discussion by backun** [botman/botman#1372] [Q&A]
Hi,
in a Laravel project I would like to integrate the BotMan Web Widget. I need to lead the conversation in a well-defined path, so I will use buttons to determine specific actions. I would like to know is there any way to disable the user input text so that he has to select input only from the available buttons and of course when needed also the possibility to enable the user input text again ? 
Thanks !

**Comment by filippotoso** [comment]
You can achieve what you need customizing the Web Widget code: https://github.com/botman/web-widget
